content/building-blocks/automate-and-execute-your-work/reproducible-work/google_cloud_docker.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Import and Run a python environment on Google cloud with Docker" 
+description: "Learn how to import and run a python environment on google cloud with docker"
+keywords: "docker, images, container, environments, virtual, Python, Jupyter, Jupyter notebook, environment, google cloud, cloud"
+weight: 2
+author: Diego Sanchez Perez
+draft: true
+aliases:
+
+
+---
+
+{{% warning %}}
+
+## Some comments on the draft
+
+- I have reproduced the steps I carried out for the docker assignment sent to me as basis for this draft. Though I learned a lot more about Docker during the assignment completion than what is reflected in this draft, I kept it close to the practical steps of being able to run a Python env. from a github repo that alredy provided the Dockerfile and Docker-compose without going too much into details.
+
+- For this building block it has been asumed that the project environment to replicate already contained a dockerfile and docke-compose file in the first place. From this assumption, a reasonable adition to the building block could be a brief explanation on how to interpret a dockerfile and/or a dockercompose file.
+
+- The focus of the building block could be easily reversed from import other's work to export our work if it is extended to include how to elaborate our own dockerfile though I am aware this may be too much information for a single building block. 
+
+- A brief contextualization on docker may be of value for the building block though at the same this is also covered to some extent on other extisting building blocks so instead links to these could be added.
+
+- Potential alternative: Docker DevEnvironments
+
+{{% /warning %}}
+
+# Import a Python environment from Github using Docker
+
+Take advantage of the versatility of containerized apps on Docker to make your work easier to share and reporduce! 
+
+{{% summary %}}
+
+In this building block you will learn step by step how to import a Python environment from an existing project located in a Github repository using Docker.

```

Comment:
Not really descriptive of the building block taking into account the content it will cover with all the next additions, this definitively has to be upgraded